### ISSUE TITLE
Y-axis allignment

### ISSUE THREAD

**Issue by tracs** [chartist-js/chartist#625]
I'm scratching my head what I'm doing wrong but I cannot find the issue.
My y-axis is not alligned correctly. In the fiddle it starts as it should at 5.
On localhost it reads 4.5 when I make my browser window small..
When i resize the viewport to fullsize the y axis reads 4.875.

Fiddle (ok)
http://jsfiddle.net/bpcdka75/

My code (probably not ok)

```
<html>
<head>
<!-- Required JS -->
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.0/jquery.min.js"></script>
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/chartist/0.7.3/chartist.min.css">
    <script src="https://rawgit.com/gionkunz/chartist-js/master/dist/chartist.min.js"></script>
    <script>
        $(document).ready(function(){
            var data = {
              // A labels array that can contain any sort of values
              labels: ['Mon', 'Tue', 'Wed', 'Thu', 'Fri'],
              // Our series array that contains series objects or in this case series data arrays
              series: [
                [5, 2, 4, 2, 0]
              ]
            };

            // Create a new line chart object where as first parameter we pass in a selector
            // that is resolving to our chart container element. The Second parameter
            // is the actual data object.
            new Chartist.Line('.ct-chart', data);
        });
    </script>
</head>
<body>
   <div class="ct-chart ct-square"></div>
</body>
</html>
```

And the chart
small viewport : http://imgur.com/kkTT1c7
fullsize: http://imgur.com/qR6KAUP

Does anybody know what I'm doing wrong?

**Comment by gionkunz** [comment]
You're using a very old version of the Chartist CSS locally :-) You can use the latest version from the CDN:

``` html
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js"></script>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" rel="stylesheet" type="text/css" />
```

**Comment by tracs** [comment]
Oops.....
Thanks..
